doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -337,7 +337,9 @@ DFU libraries
 Modem libraries
 ---------------
 
-|no_changes_yet_note|
+* :ref:`nrf_modem_lib_readme`:
+
+  * Fixed an issue with the CFUN hooks when the Modem library is initialized during ``SYS_INIT`` at kernel level and makes calls to the :ref:`nrf_modem_at` interface before the application level initialization is done.
```

Comment:
Checked with the team and we could add a sentence that : _This issue is also fixed in the |NCS| v2.6.0._ , since the issue is now backported to the 2.6 branch.
This could be a way to inform the customer about the fix.

Reply:
Shouldn't the backport take care of this being in the v2.6.1 changelog when it is backported? Note that it is not backported to v2.6.0, but to the v2.6-branch, so it will be in v2.6.1.

Reply:
okay if this is part of 2.6.1 release, then, this entry is not needed. Thanks!.